nrf_security/src/zephyr/CMakeLists.txt

```diff
@@ -97,10 +97,6 @@ if(CONFIG_BUILD_WITH_TFM)
     add_dependencies(${mbedcrypto_target} tfm_mbedtls_headers_copy)
   endif()
 
-  zephyr_include_directories(
-    ${ZEPHYR_TRUSTED_FIRMWARE_M_MODULE_DIR}/trusted-firmware-m/interface/include
```

Comment:
> Instead rely on the build system to make the headers available.

Good principle.
I wonder if you could refer to a merged commit or PR that has added this so we are guaranteed that removing this line will work.

Reply:
CI will guarantee it, no?

Reply:
@tejlmand Yes, I just haven't linked them together yet because the dependency is too long, going all the way to upstream TF-M. When I take this out of draft it will have the reference to the required zephyr change.

Reply:
> CI will guarantee it, no?

most likely, but that doesn't tell what and where something was changed that makes this change safe.

Reply:
Ah, I thought you meant in a comment.

Git commit message update would be good.